PukingPredator/Assets/Scripts/Consumable/Consumable.cs

```diff
@@ -294,4 +327,22 @@ private void UpdateBeingConsumed()
         var hasBeenConsumed = gameObject.transform.localScale.magnitude / initialScale.magnitude < consumptionCutoff;
         if (hasBeenConsumed) { SetState(ItemState.inInventory); }
     }
+
+    private void UpdateProximityOutline()
```

Comment:
not sure if this is a huge issue, but right now all the proximiate items are outlined not just the one you will consume if you click.